chsdi/locale/de/LC_MESSAGES/chsdi.po

```diff
@@ -4517,6 +4520,9 @@ msgstr "Datum"
 msgid "date_time"
 msgstr "Datum/Zeit"
 
+msgid "datenstand"
+msgstr "Datenstand"
```

Comment:
we probably already have this translation, please check it out first before adding new ones.